lib/crunch64/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+# Version should be synced with lib/Cargo.toml and lib/pyproject.toml
+__version_info__ = (0, 1, 0)
+__version__ = ".".join(map(str, __version_info__))
+__author__ = "Capybara man" # TODO: Ethan update this before release plz
```

Comment:
What do we want to put here?

Reply:
I think decompals is fine, or if you want to put our names that makes sense too! Up to you